src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -26,46 +37,144 @@ const containersHtml = renderToHtml(Containers);
 import Colors from './colors';
 const colorsSource = require('!!raw-loader!./colors');
 const colorsHtml = renderToHtml(Colors);
+const colorsSnippet = `<EuiComboBox
+  placeholder="Select or create options"
+  options={[
+    {
+      label: 'Titan',
+      color: "#ff0000",
+    },
+  ]}
+  onChange={onChange}
+  onCreateOption={onCreateOption}
+  isClearable={true}
+  isDisabled
+/>`;
 
 import RenderOption from './render_option';
 const renderOptionSource = require('!!raw-loader!./render_option');
 const renderOptionHtml = renderToHtml(RenderOption);
+const renderOptionSnippet = `<EuiComboBox
+  placeholder="Select or create options"
+  options={options}
+  selectedOptions={selectedOptions}
+  onChange={onChange}
+  onCreateOption={onCreateOption}
+  renderOption={renderOption}
+/>`;
 
 import Groups from './groups';
 const groupsSource = require('!!raw-loader!./groups');
 const groupsHtml = renderToHtml(Groups);
+const groupsSnippet = `<EuiComboBox
+  placeholder="These options are grouped"
+  options={[colorGroup, soundGroup]}
+  selectedOptions={selectedOptions}
+  onChange={onChange}
+  onCreateOption={onCreateOption}
+/>`;
 
 import SingleSelection from './single_selection';
 const singleSelectionSource = require('!!raw-loader!./single_selection');
 const singleSelectionHtml = renderToHtml(SingleSelection);
+const singleSelectionSnippet = `<EuiComboBox
+  placeholder="Select a single option"
+  singleSelection={{ asPlainText: true }}
+  options={options}
+  selectedOptions={selectedOptions}
+  onChange={onChange}
+  isClearable={false}
+/>`;
 
 import DisallowCustomOptions from './disallow_custom_options';
 const disallowCustomOptionsSource = require('!!raw-loader!./disallow_custom_options');
 const disallowCustomOptionsHtml = renderToHtml(DisallowCustomOptions);
+const disallowCustomOptionsSnippet = `<EuiComboBox
+  placeholder="Select from a list of options"
+  options={options}
+  onChange={onChange}
+  onSearchChange={onSearchChange}
+/>`;
 
 import CustomOptionsOnly from './custom_options_only';
 const customOptionsOnlySource = require('!!raw-loader!./custom_options_only');
 const customOptionsOnlyHtml = renderToHtml(CustomOptionsOnly);
+const customOptionsOnlySnippet = `<EuiFormRow
+  label="Only custom options"
+  isInvalid={isInvalid}
+  error={isInvalid ? 'Only letters are allowed' : undefined}>
+  <EuiComboBox
+    noSuggestions
+    placeholder="Create some tags (letters only)"
+    selectedOptions={selectedOptions}
+    onCreateOption={onCreateOption}
+    onChange={onChange}
+    onSearchChange={onSearchChange}
+    isInvalid={isInvalid}
+  />
+</EuiFormRow>`;
```

Comment:
```suggestion
const customOptionsOnlySnippet = `<EuiComboBox
    noSuggestions
    placeholder="Create some tags (letters only)"
    selectedOptions={selectedOptions}
    onCreateOption={onCreateOption}
    onChange={onChange}
    onSearchChange={onSearchChange}
    isInvalid={isInvalid}
  />`;
```

I think we're fine leaving form row out of the snippet. The key part we want to show is `noSuggestions`.